docs/xml-class.rst

```diff
@@ -103,17 +103,20 @@ There is no need to ever directly construct an :ref:`Xml` object: after creating
        :arg width: if provided, either an absolute (int) value, or a percentage string like "30%". A percentage value refers to the width of the specified ``where`` rectangle in :meth:`Story.place`. If this value is provided and ``height`` is omitted, the image will be included keeping its aspect ratio.
        :arg height: if provided, either an absolute (int) value, or a percentage string like "30%". A percentage value refers to the height of the specified ``where`` rectangle in :meth:`Story.place`. If this value is provided and ``width`` is omitted, the image's aspect ratio will be honored.
 
-    .. method:: add_link(link)
+    .. method:: add_link(href=None, text=None)
```

Comment:
So we have: `add_link(link)` -> `add_link(href=None, text=None)`👍 . But, does the `=None` part mean that it is an optional parameter?